pkg/testing/tools/snapshots/snapshots.go

```diff
@@ -86,6 +86,11 @@ func (sc *SnapshotsClient) FindLatestSnapshots(ctx context.Context, branches []s
 }
 
 func (sc *SnapshotsClient) findLatestSnapshot(ctx context.Context, branch string) (snapshot *version.ParsedSemVer, err error) {
+	// TODO: find a proper fix.
```

Comment:
I don't think this TODO or the `if` below is necessary
https://snapshots.elastic.co/latest/8.x.json